src/typescript-service.ts

```diff
@@ -135,26 +143,27 @@ export class TypeScriptService implements LanguageHandler {
 	 *
 	 * @param accessDisk Whether the language server is allowed to access the local file system
 	 */
-	protected initializeFileSystems(accessDisk: boolean): void {
+	protected _initializeFileSystems(accessDisk: boolean): void {
```

Comment:
I want methods to be dispatched dynamically, which means every LSP method corresponds to a method in the handler by a well-defined naming scheme (camelCase), instead of mapping everything manually. Of course, there needs to be a way to prevent private methods from being called. Underscore-prefixing is the easiest way for this. I generally think underscore-prefixes are bad coding style too, but here they have a justification.